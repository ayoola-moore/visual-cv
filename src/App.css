@import "~antd/dist/antd.css";

.wrapper {
  padding: 2% 5% 10%;
  border: 1px solid black;
  margin: 5% auto;
  width: 50%;
  min-width: 320px;
  border-radius: 5px;
}

.field-wrapper {
  margin: 5% 0;
  border-bottom: 1px solid #878282;
}

.ant-row {
  justify-content: center;
}
