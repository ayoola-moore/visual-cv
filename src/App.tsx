import React from "react";
import "./App.css";
import Form from "./components/Form/index";

function App() {
  return <Form />;
}

export default App;
