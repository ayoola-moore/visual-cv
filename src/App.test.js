import MatchMediaMock from "jest-matchmedia-mock";
import React from "react";
import renderer from "react-test-renderer";
import App from "./App";

let matchMedia;

describe("Your testing module", () => {
  beforeAll(() => {
    matchMedia = new MatchMediaMock();
  });

  afterEach(() => {
    matchMedia.clear();
  });

  it("App renders correctly", () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
