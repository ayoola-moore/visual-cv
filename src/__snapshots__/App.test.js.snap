// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Your testing module App renders correctly 1`] = `
<div
  className="wrapper"
>
  <form
    className="ant-form ant-form-horizontal"
    id="control-hooks"
    onSubmit={[Function]}
  >
    <div
      className="ant-row ant-form-item"
      style={Object {}}
    >
      <div
        className="ant-col ant-form-item-control"
        style={Object {}}
      >
        <div
          className="ant-form-item-control-input"
        >
          <div
            className="ant-form-item-control-input-content"
          >
            <button
              className="ant-btn ant-btn-primary"
              onClick={[Function]}
              type="submit"
            >
              <span>
                Submit
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </form>
</div>
`;
